.vscode/settings.json

```diff
@@ -7,5 +7,8 @@
     },
     "[javascript]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
-    }
+    },
+    "rust-analyzer.linkedProjects": [
+        "acvm-repo/acvm_js/Cargo.toml",
+    ]
```

Comment:
The issues that I think you were running into should be solved by https://github.com/noir-lang/noir/pull/4702

Reply:
I'll merge master and try it

Reply:
Ah sorry just saw this. I've merged in master now.